src/mumble/CMakeLists.txt

```
@@ -344,7 +344,13 @@ target_include_directories(mumble
 		"${3RDPARTY_DIR}/smallft"
 )
 
-find_pkg("SndFile;LibSndFile;sndfile" REQUIRED)
+if(NOT APPLE)
+	list(APPEND SNDFILE_NAMES "SndFile")
+endif()
```

We should add a comment for why this is necessary
```suggestion
if(NOT APPLE)
	# On macOS when using Homebrew, searching for "SndFile" will succeed but it will bring up an empty target
	# that then creates linker errors. Thus we only look for that name on non-macOS systems.
	list(APPEND SNDFILE_NAMES "SndFile")
endif()
```

---

I posted a comment back in #4790 with the reasoning, and have created a commit that adds a comment and fixes the commit message.